.github/workflows/tests.yml

```diff
@@ -9,8 +9,8 @@ jobs:
 
     strategy:
       matrix:
-        php: [7.1, 7.2, 7.3, 7.4, 8.0]
-        laravel: [5.6.*, 5.7.*, 5.8.*, 6.*, 7.*, 8.*]
+        php: [7.1, 7.2, 7.3, 7.4, 8.0, 9.0]
```

Comment:
PHP 9.0? I think this version does not exist yet, use 8.1

Reply:
I didn't understand the test file well. Fixed, tnx.